src/Compilers/CSharp/Portable/Binder/FixedStatementBinder.cs

```diff
@@ -27,6 +27,18 @@ override protected ImmutableArray<LocalSymbol> BuildLocals()
             if (_syntax.Declaration != null)
             {
                 var locals = new ArrayBuilder<LocalSymbol>(_syntax.Declaration.Variables.Count);
+
+                _syntax.Declaration.Type.VisitRankSpecifiers((rankSpecifier, args) =>
+                {
+                    if (rankSpecifier.Kind() != SyntaxKind.OmittedArraySizeExpression)
```

Comment:
>rankSpecifier.Kind() != SyntaxKind.OmittedArraySizeExpression [](start = 24, length = 61)

Can this condition ever be false?

Reply:
Yes. `int[] x;`

Reply:
> Yes. int[] x;

What is the ```rankSpecifier``` in this case?

---
In reply to: [264420509](https://github.com/dotnet/roslyn/pull/32843#discussion_r264420509) [](ancestors = 264420509)

Reply:
Ah, I've done something really stupid there haven't I?
😂